submissions/semenjob/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <ul class="nav__list">
+        <li class="header__list">
+          <a class="head__link" href="#">Mail</a>
+        </li>
+        <li class="header__list">
+          <a class="head__link" href="#">Images</a>
+        </li>
+      </ul>
+      <input
+        tabindex="1"
+        class="popup__checkbox"
+        type="checkbox"
+        id="popup__checkbox"
+      />
+      <label for="popup__checkbox" class="popup__checkbox__label">
+        <img
+          class="popup__icon"
+          src="./icon/popup-button.png"
+          alt="popup-button_logo"
```

Comment:
It's just ordinary text or sentence to explain what's on image.

https://www.w3schools.com/tags/att_img_alt.asp

Reply:
this is the text to pop up when the link is broken

Reply:
color for clarity) sorry